mlflow/utils/request_utils.py

```diff
@@ -41,9 +42,10 @@ def augmented_raise_for_status(response):
 
 def download_chunk(range_start, range_end, headers, download_path, http_uri):
     combined_headers = {**headers, "Range": f"bytes={range_start}-{range_end}"}
+    timeout = DOWNLOAD_CHUNK_TIMEOUT_100M * ((range_end - range_start) // 100_000_000 + 1)
```

Comment:
I think just having `DOWNLOAD_CHUNK_TIMEOUT` without the complicated math is good enough :)

Reply:
See this comment https://github.com/mlflow/mlflow/pull/9476#discussion_r1308281610

Reply:
So we can't make the same value suitable for both 500MB file download and 100MB file download

Reply:
I saw the comment. I still don't think we need to scale based on the chunk size :)

Reply:
+1

Reply:
But we're setting 5 minutes timeout for downloading a 100MB file or 500MB file at the same time?

Reply:
Isn’t the timeout per-chunk, and the chunk is a fixed size?

Reply:
Nope. For databricks_artifact_repo, chunk_size is 100MB https://github.com/mlflow/mlflow/blob/bdaf1ea7c6f55134b0b64ec54559038a2a40069f/mlflow/store/artifact/databricks_artifact_repo.py#L461C28-L461C48 For databricks_models_artifact_repo, chunk_size is 500MB https://github.com/mlflow/mlflow/blob/bdaf1ea7c6f55134b0b64ec54559038a2a40069f/mlflow/store/artifact/databricks_models_artifact_repo.py#L155C38-L155C38

Reply:
That’s still only 2 constant sizes. Also very unclear why those are different. We should make them the same. Varying timeout by size will make debugging super hard and could lead to unintended errors. I think we should use a 5 minute timeout everywhere